README.md

```diff
@@ -157,6 +157,11 @@ ratelimit:
   #  - 123.456.789.233
   #  - 456.789.90.2
 
+# DNS caching settings
+dns_cache:
+  ttl: 5 # Time-to-live for cache entries, in seconds
+  size: 10 # Optional cache size, default: 100
```

Comment:
It's just an example, the default is 100, I don't think any of the services use > 100 domains..